src/app_engine/probers.py

```diff
@@ -12,13 +12,18 @@
 import json
 import numbers
 import webapp2
+import sys
 from google.appengine.api import mail
+from google.appengine.api import memcache
 from google.appengine.api import urlfetch
 
-def send_alert_email(tag, message):
+def send_alert_email(tag, message, instance=None):
   receiver = 'tachyon-eng@google.com'
   sender_address = 'AppRTC Notification <jiayl@google.com>'
-  subject = 'AppRTC Prober Alert: ' + tag
+  subject = 'AppRTC Prober Alert'
+  if (instance != None):
```

Comment:
I linted all the new code. I was concerned about merging to the other branch given that we've linted a bunch of the files there. This change will already require some hand changes when merging.